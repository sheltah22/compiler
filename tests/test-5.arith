if false
  then 0 / 0
  else 1
