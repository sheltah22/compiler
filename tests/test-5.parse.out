(if false (/ 0 0) 1)
